Description: Use QImage.bits instead of QImage.constBits to ease backporting
 QImage.constBits was introduced in Qt 4.7.  Should be ok here to use
 QImage.bits which allows linking against the Qt in Squeeze.
Author: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2012-12-23

--- a/src/qt/notificator.cpp
+++ b/src/qt/notificator.cpp
@@ -113,7 +113,7 @@
 {
     // Convert 00xAARRGGBB to RGBA bytewise (endian-independent) format
     QImage tmp = img.convertToFormat(QImage::Format_ARGB32);
-    const uint32_t *data = reinterpret_cast<const uint32_t*>(tmp.constBits());
+    const uint32_t *data = reinterpret_cast<const uint32_t*>(tmp.bits());
 
     unsigned int num_pixels = width * height;
     image.resize(num_pixels * BYTES_PER_PIXEL);
